install_cli.php

```diff
@@ -146,7 +146,9 @@
         continue;
     }
 
-    echo "\nProcessing Module $module\n";
+    echo "\nProcessing Module $module with relationships to "
+        .implode(", ", array_keys($tidbit_relationships[$module]))
+        .".\n";
```

Comment:
We need to also handle a case when `$tidbit_relationships[$module]` is empty. In this case (when there are no relationships) the message should stay unchanged (`echo "\nProcessing Module $module\n";`)